ExtraDry/ExtraDry.Core.Tests/Internals/LinqBuilderTests.cs

```diff
@@ -68,6 +81,7 @@ public void ThenByCompatible()
         public void ThenByDescendingCompatible()
         {
             var linqSorted = SampleData.OrderByDescending(e => e.FirstName).ThenByDescending(e => e.Number).ToList();
+            var modelDescription = new ModelDescription(typeof(Datum));
```

Comment:
This looks superfluous...

Reply:
It was, I didn't delete it when I updated the test.